tests/test_server/fixtures/factories/input.py

```diff
@@ -11,6 +11,8 @@ def input_factory(**kwargs) -> Input:
         "id": input_id,
         "created_at": extract_timestamp_from_uuid(input_id),
         "operation_id": generate_new_uuid(),
+        "run_id": generate_new_uuid(),
```

Comment:
Update lineage factories to fill up these attributes instead of using random ones